tutorials/platform/services_for_ios.rst

```diff
@@ -180,9 +180,9 @@ has 6 methods:
 
 -  ``Error post_score(Variant p_score);``
 -  ``Erroraward_achievement(Variant p_params);``
--  ``Error reset_achievements();``
--  ``Error request_achievements();``
--  ``Error request_achievement_descriptions();``
+-  ``Void reset_achievements();``
```

Comment:
Why is `Void` uppercase? I'm not very familiar with ObjC++ but AFAIK it also uses `void` with a lowercase.